### PR TITLE
fix: tektonci resources volume mounts & git ssh support

### PR DESCRIPTION
**Pull request by xuzhu-591**

### CODE REVIEW

**Review by iamyeka — APPROVED** [review]

File: charts/horizon/charts/tektonci-resources/values.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ horizon:
 
       # config git ssh
       rm -rf /root/.ssh && ln -s /tekton/home/.ssh /root
-      echo -e 'StrictHostKeyChecking no' >> /root/.ssh/config
+      echo -e 'Host *\n    StrictHostKeyChecking no' >> /root/.ssh/config
```
Comment (iamyeka) [inline charts/horizon/charts/tektonci-resources/values.yaml:37]: Add ' Host *\n' is for ?

Comment (xuzhu-591) [reply charts/horizon/charts/tektonci-resources/values.yaml:37]: set StrictHostKeyChecking to no for all hosts
``` shell
=== ~/.ssh/config ===
Host url1.com
    HostName url1.com
    IdentityFile ~/.ssh/id_key1
Host url2.com
    HostName url2.com
    IdentityFile ~/.ssh/id_key2
Host *
    StrictHostKeyChecking no
```

**Review by iamyeka — APPROVED** [review]

File: charts/horizon/charts/tektonci-resources/templates/common/git-secret.yaml [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
 kind: Secret
 metadata:
-  name: git-token
-type: Opaque
-stringData:
-  git-tokens: |
-{{ toYaml .Values.gitRepos | indent 4 }}
+  name: git-secret{{ $i }}
+  annotations:
+    tekton.dev/git-0: {{ $v.url }}
```
Comment (bysph) [inline charts/horizon/charts/tektonci-resources/templates/common/git-secret.yaml:8]: git-0 -> git-$i ?

**Review by bysph — APPROVED** [review]

**Review by iamyeka — APPROVED** [review]
LGTM

**Review by iamyeka — APPROVED** [review]

**Review by work-jlsun — APPROVED** [review]
LGTM

**Pull request merged**